integration/internal/swarm/service.go

```diff
@@ -20,6 +23,67 @@ const (
 	defaultSwarmPort = 2477
 )
 
+// ServicePoll will tweak the pollSettings for `service`
+func ServicePoll(config *poll.Settings) {
+	// Override the default pollSettings for `service` resource here ...
+
+	if runtime.GOARCH == "arm64" || runtime.GOARCH == "arm" {
+		config.Timeout = 1 * time.Minute
+		config.Delay = 100 * time.Millisecond
+	}
+}
+
+// NetworkPoll will tweak the pollSettings for `network`
+func NetworkPoll(config *poll.Settings) {
+	// Override the default pollSettings for `network` resource here ...
+	config.Timeout = 30 * time.Second
+	config.Delay = 100 * time.Millisecond
+
+	if runtime.GOARCH == "arm64" || runtime.GOARCH == "arm" {
+		config.Timeout = 50 * time.Second
+	}
+}
+
+// ContainerPoll will tweak the pollSettings for `container`
+func ContainerPoll(config *poll.Settings) {
+	// Override the default pollSettings for `container` resource here ...
+
+	if runtime.GOARCH == "arm64" || runtime.GOARCH == "arm" {
+		config.Timeout = 30 * time.Second
+		config.Delay = 100 * time.Millisecond
+	}
+}
+
+// ImagePoll will tweak the pollSettings for `image`
+func ImagePoll(config *poll.Settings) {
+	// Override the default pollSettings for `image` resource here ...
+
+	if runtime.GOARCH == "arm64" || runtime.GOARCH == "arm" {
+		config.Timeout = 30 * time.Second
+		config.Delay = 100 * time.Millisecond
+	}
+}
+
+// PluginPoll will tweak the pollSettings for `plugin`
+func PluginPoll(config *poll.Settings) {
+	// Override the default pollSettings for `plugin` resource here ...
+
+	if runtime.GOARCH == "arm64" || runtime.GOARCH == "arm" {
+		config.Timeout = 30 * time.Second
+		config.Delay = 100 * time.Millisecond
+	}
+}
+
+// VolumePoll will tweak the pollSettings for `volume`
+func VolumePoll(config *poll.Settings) {
```

Comment:
I don't think we have any async volume operations, so we won't need this one.

Reply:
ditto